test/utils/goHelpers/cmd/receiptMMPTProof/main.go

```
@@ -0,0 +1,71 @@
+	receipts, err := client.BlockReceipts(ctxWithTimeout, rpc.BlockNumberOrHashWithNumber(rpc.BlockNumber(blockNumber)))
+
+	for i, receipt := range receipts {
+		fmt.Println("receipt: ", i, receipt.TxHash)
+	}
+
+	if err != nil {
+		fmt.Println("error fetching receipts: ", err)
+		return
+	}
+
+	ctxWithTimeout, _ = context.WithTimeout(context.Background(), 1000*time.Millisecond)
```

golangci-lint (1.61.0)

lostcancel: the cancel function returned by context.WithTimeout should be called, not discarded, to avoid a context leak

(govet)